documents/motions/10.2017.0 - Committees and Teams.md

```diff
@@ -32,4 +32,9 @@ The WCA will have Committees and/or Teams that have the role of Advisory Committ
       3. The WCA Board of Directors shall have the right to withdraw the appointment of a Committee Leader, if the Committee Leader does not comply with the rights and duties of Committee Leader, after advisory consultation by the WCA Ethics Committee.
          1. For the withdrawal of the WCA Ethics Committee Leader, the WCA Board of Directors will ask for advisory consultation to the WCA Disciplinary Committee.
          2. If none of these consultations is possible due to the inexistence of these Committees or because these Committees are not functional, the WCA Board of Directors will proceed without consultation.
-5. All members of the Committee shall communicate with a carbon copy (CC) to all the other members of the Committee.
+5. Each WCA Committee and Team may have one or more Senior Member(s).
```

Comment:
I think this whole section should go. Instead, we should add a line under 4.2.2. similar to it. E.g. "Promoting outstanding Committee Members to Committee Senior Members."

Reply:
Add "promoting and demoting" to 4.2.2.